test/integration/targets/incidental_inventory_aws_ec2/templates/inventory_with_constructed.yml

```diff
@@ -10,11 +10,11 @@ filters:
 keyed_groups:
   - key: 'security_groups|json_query("[].group_id")'
     prefix: 'security_groups'
-  - key: 'tags'
+  - key: 'tags1'
```

Comment:
``tags`` is from test/support/integration/plugins/inventory/aws_ec2.py, so any disallowed keys would need to be replaced in inventory plugins too so that they're usable with constructed.